libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -418,12 +418,19 @@ bool RC_Channel::read_6pos_switch(int8_t& position)
         return false;  // This is an error condition
     }
 
-    if      (pulsewidth < 1231) position = 0;
-    else if (pulsewidth < 1361) position = 1;
-    else if (pulsewidth < 1491) position = 2;
-    else if (pulsewidth < 1621) position = 3;
-    else if (pulsewidth < 1750) position = 4;
-    else position = 5;
+    if      (pulsewidth < 1231) {
```

Comment:
nitpick: too much space here

Reply:
thanks, if astyle did not catch it, it wasn't caught...fixed